.github/workflows/open-ebooks.yml

```diff
@@ -0,0 +1,92 @@
+name: Open eBooks Build & Test
+
+on: push
+# on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Reposistory
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Build
+        env:
+          NYPL_S3_ACCESS_KEY:               ${{ secrets.NYPL_S3_ACCESS_KEY }}
+          NYPL_S3_SECRET_ACCESS_KEY:        ${{ secrets.NYPL_S3_SECRET_ACCESS_KEY }}
+          MAVEN_CENTRAL_USERNAME:           ${{ secrets.MAVEN_CENTRAL_USERNAME }}
+          MAVEN_CENTRAL_PASSWORD:           ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
+          MAVEN_CENTRAL_STAGING_PROFILE_ID: 'af061f5afba777'
+          MAVEN_CENTRAL_SIGNING_KEY_ID:     'Library Simplified'
+          NYPL_GITHUB_ACCESS_TOKEN:         ${{ secrets.NYPL_GITHUB_ACCESS_TOKEN }}
+        run: ./gradlew :simplified-app-openebooks:assembleRelease --stacktrace
```

Comment:
@io7m are there build configurations in place that will cause this resulting APK to be located somewhere other than: `simplified-app-openebooks/build/outputs/apk/release`? Running this assemble command locally puts the APK there, but from on this Github action, the APK is not located there, the build directory is created either.